tests/PHPUnit/Fixtures/UITestFixture.php

```diff
@@ -99,11 +99,11 @@ public function setUp(): void
         SitesManagerAPI::getInstance()->updateSite(1, null, null, true);
 
         // create non super user
-        UsersManagerAPI::getInstance()->addUser('oliverqueen', 'smartypants', 'oli@queenindustries.com');
+        UsersManagerAPI::getInstance()->addUser('oliverqueen', 'smartypants', $this->xssTesting->forTwig('useremail').'@queenindustries.com');
```

Comment:
Actually I'm not sure if those emails are visible anywhere in the UI tests. But it seems the generated email is actually not valid. See https://app.travis-ci.com/github/matomo-org/matomo/jobs/539493564

To test some special emails in the UI it might be good to also change another fixture maybe, so the addresses are visible in the UsersManager UI tests:
https://github.com/matomo-org/matomo/blob/4.x-dev/plugins/UsersManager/tests/Fixtures/ManyUsers.php